pom.xml

```diff
@@ -55,6 +55,16 @@ limitations under the License.
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-api-xml</artifactId>
+      <version>4.0.0-alpha-4</version>
```

Comment:
I find it weird to depend on a version 4.0.0-alpha-4
I already fear the dependabot upgrades in the future when new Maven releases happen